cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java

```diff
@@ -226,10 +226,8 @@ public void onBackPressAction() {
     }
 
     public void onAction(ConsentAction action) {
-        if (GDPRConsentLib.this.onAction != null)
-            uiThreadHandler.postIfEnabled( () ->GDPRConsentLib.this.onAction.run(action.actionType));
-
         try {
+            uiThreadHandler.postIfEnabled( () ->GDPRConsentLib.this.onAction.run(action.actionType));
```

Comment:
Here we may get NPE if application implementing SDK has not initialised onAction callback

Reply:
the builder initialises it as a noop callback